rocketpy/simulation/flight_data_importer.py

```diff
@@ -0,0 +1,196 @@
+"""Starts with .csv or .txt file containing the rocket's collected flight data
+and build a rocketpy.Flight object from it.
+"""
+import warnings
+
+import numpy as np
+
+from rocketpy.mathutils import Function
+from rocketpy.units import UNITS_CONVERSION_DICT
+
+FLIGHT_LABEL_MAP = {
+    # "name of Flight Attribute": "Label to be displayed"
+    "time": "Time (s)",
+    "x": "Position x (m)",
+    "y": "Position y (m)",
+    "z": "Position z (m)",
+    "vx": "Velocity x (m/s)",
+    "vy": "Velocity y (m/s)",
+    "vz": "Velocity z (m/s)",
+    "ax": "Acceleration x (m/s^2)",
+    "ay": "Acceleration y (m/s^2)",
+    "az": "Acceleration z (m/s^2)",
+    "altitude": "Altitude AGL (m)",
+    "latitude": "Latitude (deg)",
+    "longitude": "Longitude (deg)",
+}
+
+
+class FlightDataImporter:
+    """A class to create a rocketpy.Flight object from a .csv file. The data
+    frame must contain a time column in seconds.
+
+    There are different ways of calculating a property from the data frame.
+        1. The property is already in data frame and you want to use it as is
+        2. The property is not in the data frame but you can calculate it from
+            other properties in the data frame.
+
+    Initially, this class is designed to work with the option '1.'
+
+    Parameters
+    ----------
+    name : str
+        The name of the flight. This is useful to identify the flight in the
+        simulation. Example: "Rocket Calisto, Spaceport America, 2019"
+    filepath : str, Path
+        The path
+    columns_map : dict, optional
```

Comment:
I wonder this could be a list of filepaths and, as long as they have headers and a "time" column, it should be easy to make it work

Reply:
Good suggestions, done!